cantabular/queries.go

```diff
@@ -116,6 +116,30 @@ query($dataset: String!) {
 	}
 }`
 
+const QueryDimensionCategories = `
+query ($dataset: String!, $variables: [String!]!) {
+  dataset(name: $dataset) {
+    variables(names: $variables) {
+      edges {
+	node {
+	  name
+	  label
+	  categories {
+	    edges {
```

Comment:
@ChrysmOre  done, used slightly less spaces when formatting cause its easier to read, the practice of tabs for formatting makes for very wide code